src/components/JobCardComponent.jsx

```diff
@@ -0,0 +1,108 @@
+import React from "react";
+import styled from "@emotion/styled";
+import scootLogo from "../assets/images/logos/scoot.svg";
```

Comment:
I am not sure how to do the dynamic imports without using state. Since we are storing the images in src, we can't directly use the path in image src as just a variable from json.

Reply:
we also need to import the url in componenet level to access the image and the approach just passing it as props obly works if they are from the public directory

Reply:
Great catch! Can you please create a separate PR to move all the company logos into the `public` directory and update the image URL paths in the `data.json` file? Thank you for bringing up this concern!